tests/ui/flatten_and_methods.stderr

```diff
@@ -1,4 +1,4 @@
-error: methods and doc comments are not allowed for flattened entry
+error: methods are not allowed for flattened entry
```

Comment:
Wasn't the goal to ignore the doc  comment? Here, the compilation fail.

Reply:
The goal is to ignore the comments, but disallow methods with flatten, like `short`. This is exactly what triggers it here. 
```
/// Comments allowed
#[structopt(flatten)]

// Methods disallowed
#[structopt(flatten, short)]
```